src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -308,6 +309,12 @@ public void periodic() {
       logDesiredPosition.append(desiredState.position);
       logDesiredVelocity.append(desiredState.velocity);
       logCurrentVoltage.append(currentVoltage);
+    } else {
+      if (goalState.position == MIN_HEIGHT
+          && !MathUtil.isNear(MIN_HEIGHT, currentState.position, STOWED_POSITION_TOLERANCE)
+          && currentState.velocity == 0) {
```

Comment:
`&& currentState.velocity == 0` Should we add a range here? double variables / encoder readings rarely exactly equals 0
```suggestion
private static final double STUCK_VELOCITY_TOLERANCE = 0.01; 
          && Math.abs(currentState.velocity) < STUCK_VELOCITY_TOLERANCE) {
```

Reply:
More generally for "exact" floating point value comparisons, we should use an [epsilon](https://codingtechroom.com/question/java-double-epsilon) to check the value. The motivation for this is due to precision limitations in floating-point values. However, `0.0` can be exactly represented and we do see the `velocity` reported as zero on the Shuffleboard. Still, it would be reasonable to apply the epsilon check here I suppose. 1 cm/s is far too large a tolerance, though.